py/specs/count.toml

```diff
@@ -0,0 +1,21 @@
+[setup]
+statement_files = ["sql/uservisits.sql"]
+statements = [
+    "copy uservisits from 's3://crate.amplab/data/tiny/uservisits/part-*' with (compression = 'gzip')",
```

Comment:
i think we should use the 1node dataset instead of the tiny dataset to decrease volatility